pkg/sidecar/config.go

```diff
@@ -0,0 +1,62 @@
+package sidecar
+
+import (
+	"errors"
+	"os"
+
+	"github.com/hashicorp/hcl"
+	"github.com/spiffe/go-spiffe/v2/logger"
+)
+
+// Config contains config variables when creating a SPIFFE Sidecar.
+type Config struct {
+	AgentAddress string `hcl:"agentAddress"`
+	Cmd          string `hcl:"cmd"`
+	CmdArgs      string `hcl:"cmdArgs"`
+	CertDir      string `hcl:"certDir"`
+	// Merge intermediate certificates into Bundle file instead of SVID file,
+	// it is useful is some scenarios like MySQL,
+	// where this is the expected format for presented certificates and bundles
+	AddIntermediatesToBundle bool   `hcl:"addIntermediatesToBundle"`
+	SvidFileName             string `hcl:"svidFileName"`
+	SvidKeyFileName          string `hcl:"svidKeyFileName"`
+	SvidBundleFileName       string `hcl:"svidBundleFileName"`
+	RenewSignal              string `hcl:"renewSignal"`
+	// TODO: is there a reason for this to be exposed? and inside of config?
+	ReloadExternalProcess func() error
```

Comment:
I added this in a long time ago. We were importing the `github.com/spiffe/spiffe-helper/pkg/sidecar` into our go program to save certificates to disk and call a function to reload process instead of sending a signal.

Reply:
do you think that code is still in use?
is something we can remove in a near future?

Reply:
I think so, we ended up not using it.